gcp_variant_transforms/libs/variant_partition.py

```diff
@@ -34,59 +44,266 @@
 # will be matched to next available partitions [22, 27).
 _DEFAULT_NUM_PARTITIONS = _RESERVED_AUTO_PARTITIONS + 5
 
+# At most 1000 partitions can be set as output of VariantTransform.
+_MAX_NUM_PARTITIONS = 1000
+# Each partition can contain at most 64 regions.
+_MAX_NUM_REGIONS = 64
+# Matches to regions formatted as 'chr12:10,000-20,000' used in par_config.yaml
+_REGION_LITERAL_REGEXP = re.compile(r'^(\S+):([0-9,]+)-([0-9,]+)$')
+# A special literal for identifying residual partition's region name.
+_RESIDUAL_REGION_LITERAL = 'residual'
+PARTITION_NOT_FOUND = -1
```

Comment:
How about UNDEFINED_PARTITION_INDEX? PARTITION_NOT_FOULD looks like a boolean value. We're also using it to initialize for `_residual_partition_index`